File: HWC24/outros/Python_Projetos/cursoEmVideo/aula7_OperadoresAritmeticos/exercicio008.py
entrada = input("Uma distância em metros: ")
print("A medida de {:.1f}m corresponde a".format(float(entrada))
      ,"\n{}km\n{}hm\n{}dam\n{}dm\n{}cm\n{}mm\n"
      .format(float(entrada)/1000, float(entrada)/100, 
              float(entrada)/10, float(entrada)*10, 
              float(entrada)*100, float(entrada)*1000))

File: HWC24/outros/Python_Projetos/cursoEmVideo/aula7_OperadoresAritmeticos/exercicio008.c
#include <stdio.h>

int main(){
    float entrada = 0; 
    printf("Uma distancia em metros: ");
    scanf("%f", &entrada);

    printf("A medida de %.1fm corresponde a \
            \n%.4fkm\n%.3fhm\n%.2fdam\n%.1fdm\n%.1fcm\n%.1fmm\n"\
            , entrada, entrada/1000, entrada/100, entrada/10,\
            entrada*10, entrada*100, entrada*1000);
}